src/index.js

```diff
@@ -33,25 +50,31 @@ class CyclicS3FS {
   }
 
   async writeFile(fileName, data, options={}) {
+    if (this.isLocal){
```

Comment:
instead of doing all these ifs everywhere, just return regular fs from the constructor right after the check

Reply:
can't really do it in the constructor actually but maybe a static method/factory

Reply:
👍